examples/basic_infection/README.md

```diff
@@ -25,6 +22,20 @@ Infected individuals schedule their recovery at time `t + infected period`. The
 flowchart LR
 S(Susceptible) --FoI--> I(Infected) --inf. period--> R(Recovered)
 ```
+## Simulation architecture
+The simulation executaion first initializes the model parameters and prepares the simulation modules.
+
+### Initialization and modules
```

Comment:
I think it would be good for each of these modules to flesh out some more detail - enough so an engineer could assess what features Context needs to have, and to even be able to make good progress translating your outline to code.

So, you could describe what each module does upon initialization (if anything) in terms of registering for events, making plans, etc. Then for each of the function/closure handlers for those events and plans you should describe what happens in the body of that function/closure.

Reply:
To follow on here, I think it's useful to understand the flow of data.

So, for instance, the transmission manager infects person X, and then the infection manager has to do something, right? How does that happen? I would also add reporting, as it illustrates some other pieces, especially if you want to do status reporting, not event reporting.

Reply:
I elaborated a bit more with transmission and infection managers. I think they are a bit crude in the sense that some of the interactions with the context are still unclear. I will add more detail to the other modules but pushed for now to get some feedback.

Reply:
Also, yeah, I will draft something for the reporting module soon.